content/en/docs/pluggable-components/auditing-logs.md

```diff
@@ -7,157 +7,144 @@ linkTitle: "KubeSphere Auditing Logs"
 weight: 3525
 ---
 
-## What are KubeSphere Auditing Logs?
+## What are KubeSphere Auditing Logs
 
 KubeSphere Auditing Log System provides a security-relevant chronological set of records documenting the sequence of activities related to individual users, managers, or other components of the system. Each request to KubeSphere generates an event that is then written to a webhook and processed according to a certain rule.
 
-For more information, see Logging, Events, and Auditing.
+For more information, see [Auditing Log Query](../../toolbox/auditing/auditing-query).
 
 ## Enable Auditing Logs before Installation
 
 ### Installing on Linux
 
-When you install KubeSphere on Linux, you need to create a configuration file, which lists all KubeSphere components.
+When you implement multi-node installation KubeSphere on Linux, you need to create a configuration file, which lists all KubeSphere components.
```

Comment:
implement multi-node installation KubeSphere
-> 
implement multi-node installation of KubeSphere